### PR TITLE
use passAsFile for writeTextFile if available

### PR DESCRIPTION
**Pull request by phunehehe**
tagged [mass-rebuild](https://github.com/NixOS/nixpkgs/labels/mass-rebuild) ;)

### CODE REVIEW

File: nixos/cloud-config.nix [hunk 1]
```diff
@@ -16,6 +16,7 @@ in
   time.timeZone = cloudDefault "UTC";
   i18n.supportedLocales = cloudDefault ["en_US.UTF-8/UTF-8"];
 
+  nix.package = pkgs.nixUnstable;
```
Comment (proger) [inline nixos/cloud-config.nix:19]: i think we need to backport the expression for `nixUnstable`

Comment (proger) [inline nixos/cloud-config.nix:19]: or not?

Comment (phunehehe) [inline nixos/cloud-config.nix:19]: nope, it just works (tm)

**Comment by proger** [comment]
Thanks!

**Pull request merged**